tool_quality_checks/test/test_scripts.py

```diff
@@ -65,6 +65,7 @@ def test_resolve_ini_error():
 pipe_layer_abs_path = os.path.join(OUR_DIR, _pipe_layer_rel_path)
 _raster_rel_path = "data\schiedam-test\dem_schiedam_test.tif"
 raster_abs_path = os.path.join(OUR_DIR, _raster_rel_path)
+geopackage_abs_path = os.path.join(OUR_DIR, "data\schiedam-test\export_scripts.gpkg")
```

Comment:
Backslashes in slashes veranderen. 
(En nog op een paar andere plekken, dus even de zoekfunctie in je editor gebruiken).
Nu gaat \s en \e toevallig goed, maar "\n" gaat problemen geven, bijvoorbeeld.